### PR TITLE
Bump Oasis SDK to rev b8e5f8f8c725eaf28f643f221930b55775b5a765

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -47,7 +47,7 @@ impl sdk::Runtime for Runtime {
     const VERSION: Version = sdk::version_from_cargo!();
     /// Current version of the global state (e.g. parameters). Any parameter updates should bump
     /// this version in order for the migrations to be executed.
-    const STATE_VERSION: u32 = 5;
+    const STATE_VERSION: u32 = 6;
```
Comment (pro-wh) [inline runtime/src/lib.rs:50]: how does this change work? no migrations, only bump to prevent downgrading?

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -21,8 +21,8 @@ debug = false
 cipher-keymanager = { path = "../keymanager" }
 
 # SDK.
-oasis-runtime-sdk = { git = "https://github.com/oasisprotocol/oasis-sdk", rev = "cca69ecc53bfa3909198aae5ed4f7af6662cd39b" }
-module-contracts = { git = "https://github.com/oasisprotocol/oasis-sdk", rev = "cca69ecc53bfa3909198aae5ed4f7af6662cd39b", package = "oasis-runtime-sdk-contracts" }
+oasis-runtime-sdk = { git = "https://github.com/oasisprotocol/oasis-sdk", rev = "b8e5f8f8c725eaf28f643f221930b55775b5a765" }
+module-contracts = { git = "https://github.com/oasisprotocol/oasis-sdk", rev = "b8e5f8f8c725eaf28f643f221930b55775b5a765", package = "oasis-runtime-sdk-contracts" }
```
Comment (pro-wh) [inline runtime/Cargo.toml:25]: :information_source: cross reference https://github.com/oasisprotocol/oasis-sdk/commit/b8e5f8f8c725eaf28f643f221930b55775b5a765

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -84,8 +84,8 @@ impl sdk::Runtime for Runtime {
         if is_testnet() {
             // Testnet.
             Some(TrustRoot {
-                height: 9500687,
-                hash: "9cdd98b0acef04edf3056ce0b66b8323474f8c0048ad15ea6f97c3a4fc27a1d2".into(),
+                height: 11379296,
+                hash: "4a854864c82e3214bf560377d7a84650402ad0f5f4d2097f22f89757f95c6ade".into(),
```
Comment (pro-wh) [inline runtime/src/lib.rs:88]: :information_source: that block on oasis scan https://testnet.oasisscan.com/blocks/11379296

**Review by pro-wh — APPROVED** [review]

File: runtime/src/lib.rs (hunk 1 above)
Comment (kostko) [reply runtime/src/lib.rs:50]: Bump to trigger a migration that will update the parameters.

Comment (pro-wh) [reply runtime/src/lib.rs:50]: oh right that new gas parameter

**Pull request merged**